### ISSUE TITLE
Export model in Torchscript format

### ISSUE THREAD

**Issue by Grace6666** [open-mmlab/mmdetection#4863]
It seems ONNX format is supported for faster rcnn, yolo and ssd. Is it possible to export these models to TorchScript format.

**Comment by ZwwWayne** [comment]
Yes, we are working on that and this feature may be supported in future releases. Please stay tuned. PRs are also welcomed.

**Comment by ake020675** [comment]
how's it going now

**Comment by austinmw** [comment]
Any updates on torchscript? Is it possible to use `torch.jit.script` and then run `mmdet2torchserve.py` on a scripted model?

**Comment by congjianting** [comment]
Any news for torchscript?
